plugin/evm/staged_timer.go

```diff
@@ -1,20 +0,0 @@
-// Copyright (C) 2019-2021, Ava Labs, Inc. All rights reserved.
-// See the file LICENSE for licensing terms.
-
-package evm
-
-import "time"
-
-// NewStagedTimer returns a timer that will execute [f]
-// when a timeout occurs and execute an additional timeout after
-// the returned duration if [f] returns true and some duration.
-func NewStagedTimer(f func() (time.Duration, bool)) *Timer {
-	t := NewTimer(nil)
```

Comment:
curious to why we deleted these?

Reply:
and all timer related code?

Reply:
We migrated this in during a broken build and created a ticket to remove it when possible: https://github.com/ava-labs/subnet-evm/issues/263.

This was used in the previous version of the block builder logic, and since that has now been removed it's dead code.